playgame.py

```diff
@@ -18,12 +19,25 @@
 
 def main(argv):
     args = parse_args()
+    print ("asd")
     prep_logger(args.log_dir, args.verbose, args.log_stderr, len(args.players))
     if args.is_tournament:
         run_tournament(args)
     else:
         run_one_game(args)
 
+def create_dirs(default,path):
+    default_file = default
+    path = path
+    path = default_file +"/" + path
+    list = path.split("/")
+    directory = ""
+    for dirs in list[:-1]:
```

Comment:
getting on a flight so I'm looking at this quickly. What's going on here? you don't look at the last element since you know its the filename? this seems overly complex...

Reply:
My idea was to create a default root directory called "results" for log files. In this section I am creating subdirectories for case when user input is like  --log_dir "a/b/results.txt" function creates a directory called "a" in default directory "results" and dir. "b" in dir. "results/a" and the file results.txt will have the path: results/a/b/results.txt